example/nodejs-assets/build-native-modules-MacOS-helper-script-node.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+      # Helper script for Gradle to call node on macOS in case it is not found
+      export PATH=$PATH:/var/folders/mt/ylvdz3ld5b16ck4gtw6g663w0000gn/T/yarn--1686055109490-0.9487245569392035:/Users/luca/Sites/gr4vy-react-native/example/node_modules/nodejs-mobile-react-native/node_modules/.bin:/Users/luca/.config/yarn/link/node_modules/.bin:/Users/luca/.yarn/bin:/Users/luca/.nvm/versions/node/v16.17.0/libexec/lib/node_modules/npm/bin/node-gyp-bin:/Users/luca/.nvm/versions/node/v16.17.0/lib/node_modules/npm/bin/node-gyp-bin:/Users/luca/.nvm/versions/node/v16.17.0/bin/node_modules/npm/bin/node-gyp-bin:/Users/luca/.rbenv/shims:/Users/luca/.jenv/shims:/Users/luca/.jenv/bin:/Users/luca/Library/Android/sdk/cmdline-tools/latest/bin:/Users/luca/Library/Android/sdk/emulator:/Users/luca/Library/Android/sdk/platform-tools:/Users/luca/.nvm/versions/node/v16.17.0/bin:/Users/luca/.local/bin:/Users/luca/.pyenv/shims:/Users/luca/.pyenv/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Library/Apple/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
```

Comment:
Should this file be commited with absolute paths `Users/luca` ?
Running `yarn` will change this file to the local users details - so looks like its a generated file.